packages/core/src/browser/storage-service.ts

```diff
@@ -76,4 +84,31 @@ export class LocalStorageService implements StorageService {
         const pathname = typeof window === 'undefined' ? '' : window.location.pathname;
         return `theia:${pathname}:${key}`;
     }
+
+    private ShowMessageNotification() {
+        const READ_INSTRUCTIONS_ACTION = 'Read Instructions';
+        const ERROR_MESSAGE = `Your prefered browser local storage is almost full.
+            To be able to save your current workspace layout or data, you may need to free some space.
+            You can refer to Theia page for instructions.
```

Comment:
There are typos in your `ERROR_MESSAGE`. 
`Your preferred browser's local storage is almost full.
To be able to save your current workspace layout or data, you may need to free up some space.
You can refer to Theia's documentation page for instructions on how to manually clean your browser's local storage.`

Reply:
Done.